src/node/core.go

```diff
@@ -296,11 +296,11 @@ func (c *Core) AddSelfEventBlock(otherHead string) error {
 	var flags int
 
 	if perr != nil {
-		return fmt.Errorf("Error retrieving parent: %s", perr)
+		fmt.Errorf("Error retrieving parent: %s", perr)
```

Comment:
Why aren't you `return`ing the error anymore?

Reply:
Sorry, overlooked when merging from another branch
I'll fix it now

Reply:
I rebase with master, now everything is ok